radicle-cli/src/commands/path.rs

```diff
@@ -2,6 +2,7 @@
 use std::ffi::OsString;
 
 use anyhow::anyhow;
+
```

Comment:
They can be grouped together, they are both external crates.

Reply:
`radicle` is a local crate, see [this](https://github.com/radicle-dev/heartwood/blob/8d8518bbab6fe847db96a7d10909f534f0981cde/radicle-cli/Cargo.toml#L31).

separating them also is helpful, cause we import a lot of radicle-specific stuff from these crates, and having them in two groups improves visibility.